docs/SRS-Volere/SRS.tex

```diff
@@ -744,15 +744,41 @@ \subsection{Adaptability Requirements}
 
 \section{Security Requirements}
 \subsection{Access Requirements}
-\lips
+\begin{enumerate} [{SR-AR}1.]
+  \item Only authorized contractors and employees should be able to access the
+    portal for completing work on site.
+  \item Only users with managerial level access should be able to get data on
+    contractors and employees.
+  \item Only users with administrative access should be able to make changes to
+    site settings.
+  \item The upload and changing of certain file classes will be tied to
+    different access levels.
+\end{enumerate}
 \subsection{Integrity Requirements}
-\lips
+\begin{enumerate} [{SR-IR}1.]
+  \item The application should not allow the addition, removal or modification
+    of files by users without proper access and authentication.
+  \item The documents displayed should match the appropriate version available
+    on SharePoint and/or MySDS.
+  \item The application should minimize the amount of bad or incomplete data
+    inputted through the portal.
+\end{enumerate}
 \subsection{Privacy Requirements}
-\lips
+\begin{enumerate} [{SR-PR}1.]
+  \item Data transmission will be encrypted.
+  \item The application should protect private information in accordance with
+    any and all government privacy laws and policies.
+\end{enumerate}
 \subsection{Audit Requirements}
-\lips
+\begin{enumerate} [{SR-AR}1.]
+  \item The application will adhere to any and all audit rules followed by City
+    departments.
+\end{enumerate}
 \subsection{Immunity Requirements}
-\lips
+\begin{enumerate} [{SR-IMR}1.]
+  \item Security vulnerabilities and weaknesses should be patched as
+    soon they are identified.
```

Comment:
nit: we should maybe include a time frame IMO. Like expected to patch within a week or something like that.

Reply:
because we aren't going to be able to get all of them as soon as they are identified

Reply:
yep, added a timeframe